content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md

```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
 - [OWASP Mobile Top 10](https://owasp.org/www-project-mobile-top-10)
 - [OWASP Application Security Verification Standard (ASVS)](https://owasp.org/www-project-application-security-verification-standard)
 
-We look at application logic by working with your app.
+### Share Mobile App Files
+
+We look at application logic by working with your app. To support our pentesters, share the .ipa (iOS) and/or the .apk (Android) 
+files when you [Define Your Assets](/getting-started/assets/). You may also share this information in the private Slack channel
+for your pentest.
+
+### Methodology Details
 
 We follow an industry standard methodology primarily based on the OWASP Application Security
 Verification Standard (ASVS) and Testing Guide. Our team takes the following steps to ensure
 full coverage:
 
-- Reconnaissance
-  - Share the mobile application files
-    - Android: .apk
-    - iOS: .ipa
-- Automated and Manual Testing
-- Exploit Discovered Vulnerabilities
-- Report, triage, and retest
+Select the <i style="font-size:x-large; color: #0047AB" class="fas fa-chevron-right"></i> key for more information on each step.
+
+{{% expand "Target scope reconnaissance" %}}
+</br>
+Based on the Pentest Brief prepared by the client, Cobalt pentesters search for
+information about the targets and investigate the scope. This information
+includes:
+
+- Understanding workflows
+- Understanding business logic
+- Mapping the attack surface of the application
+
+Pentesters then confirm that they can:
+
+- Reach and scan the targets
+- Test the functionality of the application
+
+Our pentesters may use scanning tools such as:
+
+- MobSF
+- Frida
+- Apktool
+- Dex2Jar
```

Comment:
@mjang-cobalt Can we set up a shortcode for this? I see that this part is repeated several times.

Reply:
There are a number of sections in all of these where we should set up short codes. And while I should start now, I was having trouble thinking, "what should I name the file"... But your question is making me think again... and I realize they're all the same in this particular methodology. ding!